sda/cmd/api/api_test.go

```diff
@@ -1138,7 +1139,7 @@ func (suite *TestSuite) TestListUserFiles() {
 	testUsers := []string{"user_example.org", "User-B", "User-C"}
```

Comment:
```suggestion
	testUsers := []string{"user@example.org", "User-B", "User-C"}
```

Reply:
Nope, look at what's really happening here and which functions being called.